### ISSUE TITLE
this mod causes Dungeoncrawl spawners not to generate in 1.16.5

### ISSUE THREAD

**Issue by nodvick** [Lothrazar/unbreakable#3]
Minecraft Version: 
 1.16.5

Forge Version: 
  36.0.42

Mod Version:
 unbreakables-1.16.2-1.0.0.jar 28 aug

Single Player or Server:
 single

Describe problem (what you were doing; what happened; what should have happened):
  mod conflict with Dungeon Crawl : All Dungeon Crawl generated spawners are replaced by what natural block WOULD be in that location (stone, dirt, etc) if dungeon crawls were not installed

Log file link:
 not needed for this, no crash

Video/images/gifs (direct upload or link): 
  https://i.imgur.com/lVIMWLv.png

TL:DR - mod-generated spawners from DungeonCrawl are not generating with **_this_** mod installed

reported to dungeoncrawl as well at: https://github.com/XYROC/DungeonCrawl/issues/64
